cmd/kubectl_token.go

```diff
@@ -78,6 +77,14 @@ var supportedAuthProviders = map[string]bool{
 }
 
 func CredentialCommand() cli.Command {
+	configDir, err := ConfigDir()
+	if err != nil {
+		if runtime.GOOS == "windows" {
+			configDir = "%HOME%\\.rancher"
```

Comment:
I know this is a backport but is seems to me like this should have `os.ExpandEnv`.

Reply:
Yeah thats a good point, though in this case `configDir` is only used in the usage output for the delete subcommand. Depending on the length of `%HOME%` the usage string might get a bit longer than needed